### PR TITLE
fix: fix overlap, and guest user bugs []

### PR DESCRIPTION
**Pull request by Astitva877**
## Description
fixed previous sprint bugs 
#1091 
#1100
#1058 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

File: src/packages/@app/pages/Dashboard/Dashboard.svelte [hunk 1]
```diff
@@ -204,7 +204,7 @@
     <section class="w-100">
       <!-- Route for Collections -->
       <Route path="/collections/*">
-        <CollectionsPage />
+        <CollectionsPage {isLoginBannerActive} />
       </Route>
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/Dashboard/Dashboard.svelte:208]: not required to pass anything to collection page as  a props, as collection page has their own viewmodel to handle this.

File: src/packages/@workspaces/features/rest-explorer/layout/RestExplorer.svelte [hunk 2]
```diff
@@ -127,7 +128,12 @@
...
 {#if $tab.tabId}
-  <div class="d-flex rest-explorer-layout">
+  <div
+    class="d-flex rest-explorer-layout"
+    style="height: {isLoginBannerActive
+      ? 'calc(100vh - 126px)'
+      : 'calc(100vh - 80px)'};"
+  >
     <div class="w-100 d-flex flex-column h-100 p-3">
       <!-- Request Name Header -->
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/rest-explorer/layout/RestExplorer.svelte:138]: we have a seperate layout file for rest explorer where we maintain the layout of the rest explorer, this should be applied on that file only

File: src/packages/@app/pages/Collections/CollectionsPage.svelte [hunk 3]
```diff
@@ -259,7 +261,7 @@
         {#if $activeTab?.type === ItemType.REQUEST}
           <Motion {...scaleMotionProps} let:motion>
             <div use:motion>
-              <RestExplorerPage tab={$activeTab} />
+              <RestExplorerPage tab={$activeTab} {isLoginBannerActive} />
             </div>
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/Collections/CollectionsPage.svelte:265]: same here

Comment (Astitva877) [reply src/packages/@app/pages/Collections/CollectionsPage.svelte]: fixed

File: src/packages/@app/pages/Dashboard/Dashboard.svelte (hunk 1 above)
Comment (Astitva877) [reply src/packages/@app/pages/Dashboard/Dashboard.svelte]: fixed

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**